homeassistant/components/velbus/climate.py

```diff
@@ -19,6 +19,9 @@
 from .const import DOMAIN, PRESET_MODES
 from .entity import VelbusEntity, api_call
 
+THERMOSTAT_MODE_MAP: dict[str, HVACMode] = {}
+
+THERMOSTAT_INV_MODE_MAP = {v: k for k, v in THERMOSTAT_MODE_MAP.items()}
```

Comment:
Where do we now use this?

Reply:
that I dont know.. its part of the old code.. but removing line 22 renders the controls unusable.. and the thermostat on the HA portal stays greyed out.. I believe it is used to read the possible options from velbusIO and push those to HA.- as well as copy the controls from HA to velbusIO

Reply:
But there's nothing in home assistant that uses this and the code you added don't use this, so I'm not sure how that happens

Reply:
hmm you might be right.. I've dropped both lines.. and it seems to continue to work..